app/src/main/java/com/odinwar/mvvmdemoapp/data/model/MovieItem.kt

```diff
@@ -0,0 +1,34 @@
+package com.odinwar.mvvmdemoapp.data.model
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+data class MovieItem(
+    @SerializedName(MovieEntry.ID)
```

Comment:
e có thể đưa trực tiếp vào như thế này cho gọn nhé
```suggestion
    @SerializedName("id")
```